data/templates/frr/zebra.traffic_engineering.frr.j2

```diff
@@ -0,0 +1,37 @@
+{% if interface is vyos_defined %}
+!
+{%     set ns = namespace(mask_by_admin_group=dict()) %}
+{%     if admin_group is vyos_defined %}
+{%         for ag in admin_group %}
+{%             set _ = ns.mask_by_admin_group.update({ag: 1.__lshift__(admin_group[ag]['bit_position'] | int)}) %}
+{%         endfor %}
+{%     endif %}
+{%     set mbit_per_second_2_byte_per_second = 1024 * 1024 / 8 %}
+{%     for ifname, data in interface.items() %}
+interface {{ ifname }}
+ link-params
+{%         if 'metric' in data %}
+  metric {{ data['metric'] }}
+{%         endif %}
+{%         if 'admin_group' in data %}
+{%             set val = namespace(x=0) %}
+{%             for ag in data['admin_group'] %}
+{%                 set val.x = (val.x | int).__or__(ns.mask_by_admin_group[ag]) %}
+{%             endfor %}
+  admin-grp {{ '0x{:x}'.format(val.x) }}
+{%         endif %}
+{%         if 'max_bandwidth' in data %}
+  max-bw {{ (data['max_bandwidth'] | int) * mbit_per_second_2_byte_per_second }}
```

Comment:
This style would be coinsistent with the rest of the file - is there a reason to not use it?

```suggestion
  max-bw {{ (data.max_bandwidth | int) * mbit_per_second_2_byte_per_second }}
```